spec/rspec/snapshot/matchers_spec.rb

```diff
@@ -1,15 +1,21 @@
-require "spec_helper"
-require "json"
+require 'spec_helper'
+require 'json'
 require 'active_support/core_ext/string'
 
 describe RSpec::Snapshot::Matchers do
-  it "snapshot json" do
-    json = JSON.pretty_generate({ a: 1, b: 2 })
+  it 'snapshot json' do
+    json = JSON.pretty_generate(a: 1, b: 2)
 
-    expect(json).to match_snapshot("snapshot/json")
+    expect(json).to match_snapshot('snapshot/json')
```

Comment:
TODO: when a name is _not_ passed, it would be awesome if the test name were used (a la [jest](https://facebook.github.io/jest/docs/en/snapshot-testing.html))